uint8 response_mode #= 3 # KUKA_RMODE_STREAM
uint32 response_ms #= 20 # [ms]
uint32 traj_id #= 666 # a trajectory id
bool run #= 1 # YES
uint32 vel #= 200 # [mm/s]
uint32 tol #= 20 # [mm]
uint8 frame_type #= 1 # KUKA_CARTESIAN
